Makefile

```diff
@@ -6,7 +6,7 @@ test:
 	pipenv run tox -p auto
 
 ci:
-	pipenv run pytest --cov=stripe -n 8
+	pipenv run pytest --cov=stripe
```

Comment:
I had to disable parallel test execution on Travis, otherwise it'd run into errors when starting a stripe-mock process and running tests in parallels. It's not a big deal since tests are still very fast (~10 seconds with coverage).

Reply:
Doh! I assume you looked into it, but as long as you could get just the stripe mock startup process to run *non*-parallel, stripe-mock can take the parallel traffic just fine.

(And just thinking about it a little more ... this will probably be an issue for stripe-go's implementation as well.)

Reply:
To be clear, I don't think the issue had to do with stripe-mock itself but rather with resource exhaustion on Travis. The errors were inconsistent, but I saw a few "Cannot allocate memory" messages. I didn't try it, but maybe reducing the number of workers would have worked.